atom/browser/ui/views/menu_bar.cc

```diff
@@ -53,9 +53,11 @@ MenuBar::MenuBar()
   UpdateMenuBarColor();
   SetLayoutManager(new views::BoxLayout(
       views::BoxLayout::kHorizontal, 0, 0, 0));
+  menu_delegate_ = new MenuDelegate(this);
 }
 
 MenuBar::~MenuBar() {
+  if (menu_delegate_) delete menu_delegate_;
```

Comment:
I'll merge, but can you make this a `std::unique_ptr` in a follow-up? Most things that are tied to the lifetime of a class should be unique_ptrs so they will be automatically deleted with the parent class. If another class needs access to it you can either return a bare ptr (with `.get()`) if the potential access can't outlive the owner or a WeakPtr if it can